packages/runtime/src/types.d.ts

```diff
@@ -78,6 +75,16 @@ export interface ComponentContext {
   env: ToddleEnv
 }
 
+export type ContextApi = Record<
+  string,
+  {
+    fetch: Function
+    destroy: Function
+    update?: Function
+    triggerActions?: Function
+  }
+>
```

Comment:
This would make it easier to work with the type 🤞 
```suggestion
export interface ContextApi = {
    fetch: Function
    destroy: Function
    update?: Function
    triggerActions?: Function
  }
```